src/frontend/src/catalog/table_catalog.rs

```diff
@@ -774,7 +793,7 @@ mod tests {
                 conflict_behavior: ConflictBehavior::NoCheck,
                 read_prefix_len_hint: 0,
                 version: Some(TableVersion::new_initial_for_test(ColumnId::new(1))),
-                watermark_columns: FixedBitSet::with_capacity(2),
+                watermark_columns: FixedBitSet::with_capacity(3),
```

Comment:
Why will watermark_columns include `rw_timestamp`

Reply:
It just sets 0 to the `FixedBitSet` and doesn't mean rw_timestamp is a watermark column.